### ISSUE TITLE
 SIGWINCH and terminal size are not propagated

### ISSUE THREAD

**Issue by fdr** [CrunchyData/bridge-cli#158]
A colleague stopped using `cb psql` because he reports that the terminal has overwrite-editing type of errors. This is unfortunate because it avoids having as many copies of database passwords in shell history, as well as checking certificates (to get around this, we add `channel_binding=require`)

I can reproduce this with terminal resizing:

1. Expand a terminal to the entire screen
2. Make it smallish
3. Type on the keys, it'll eventually improperly start wrapping the input to the beginning of the line

Example. Note `/postgres=>` is the original prompt, and my keyboard smashing has overwritten some of the usual leading text in the prompt.

```
sj ga;sklgj a;gj a;k asdfdasfafddddd/postgres=> asd;gja;ks gj;ak 
```

**Comment by abrightwell** [comment]
Thanks Dan, I'll look into it. 👍

**Comment by abrightwell** [comment]
Fixed in v3.5.1
